src/special_functions/F_eta.cc

```diff
@@ -37,38 +37,42 @@ static double leta, lgamma;
  *         \f$\frac{(x^2+2x)^{3/2}}{e^\frac{x-\eta}{\gamma}+1}\f$
  *
  * \post \c Result>=0
+ *
+ * \bug no unit tests cover this function, so commenting it out.
  */
-static double Feta_integrand(double x) {
-  double const y = x * x + 2 * x;
-  double const d = (2 * lgamma * lgamma * sqrt(2 * lgamma));
-  double const expp1 = exp((x - leta) / lgamma) + 1;
-  double const dexpp1 = -exp((x - leta) / lgamma) / lgamma;
-  double const Result = -dexpp1 * cube(sqrt(y)) / (square(expp1) * d);
+// static double Feta_integrand(double x) {
+//   double const y = x * x + 2 * x;
+//   double const d = (2 * lgamma * lgamma * sqrt(2 * lgamma));
+//   double const expp1 = exp((x - leta) / lgamma) + 1;
+//   double const dexpp1 = -exp((x - leta) / lgamma) / lgamma;
+//   double const Result = -dexpp1 * cube(sqrt(y)) / (square(expp1) * d);
```

Comment:
I cannot seem to derive this integrand for the number density at relativistic Fermi-Dirac distribution (unless assuming an unusual form for the degeneracy dependence on energy, say).  Attached to this comment is a note attempting to derive it.  Apologies for introducing any misconceptions.
Note: [fd.pdf](https://github.com/lanl/Draco/files/2387219/fd.pdf)